core/src/components/input/input.ionic.scss

```
@@ -24,10 +25,18 @@
 // ----------------------------------------------------------------
 
 .input-bottom .helper-text {
-  // TODO(FW-6112): Verify the design token is correct once it's available and remove the hardcoded value.
-  color: var(--ionic-color-neutral-600, #535353);
+  color: #{tokens.$ionic-color-neutral-600};
```

```suggestion
  color: tokens.$ionic-color-neutral-600;
```

Optional, but we don't need to escape the Sass variables when used on a style assignment.  It will compile to:

```
color: var(--ionic-color-neutral-600, #535353);
```

Pretty sure we still need to escape when binding to a CSS variable (like you are doing below).

---

Oh yeah, good catch. I just copy and pasted it 😂